### ISSUE TITLE
It would be better if allSettled will return results of effect execution

### ISSUE THREAD

**Issue by Bacher** [zerobias/effector#385]
I'm trying to write SSR Isomorphic application (with Next.js) and I start using Fork api (scopes).

As I could understand (maybe wrong) I should use `allSettled` func when I use effects on backend-side.
But If I wrap effects into `allSettled` I'm loosing my ability to read results of effect execution (returning values). And it very uncomfortable.

I expected this flavour of using:
```javascript
const user = await allSettled(getUserFx, {scope, params: { userId: 31 }});
```
but `user` will be `undefined` in this case...

For contrast calling `const user = await getUserFx({ userId: 31 };` will return user data.

Maybe in Effector exists another methods for call effects on the backend with specifying certain scope where I could get results of execution. I couldn't find.

**Comment by zerobias** [comment]
This imply that allSettled might fail (because if you get results, you should get errors too), if this is an expected behavior, then we could implement that idea

**Comment by zerobias** [comment]
Oh, looks like errors support is a breaking change, as there are projects, which expect that allSettled is never throw an error
But we can return value with type `{status: 'done', value: Done} | {status: 'fail', value: Fail}`

**Comment by Bacher** [comment]
I've thought again, and seems allSettled semantic really shouldn't return results/throw errors.
And maybe we need another method/helper for calling effects for specific scope directly.

Maybe like:
```
const user = await getUserFx.withScope(scope)({ userId: 31 });
// or
const user = await scope.exec(getUserFx, { userId: 31 });
```

But in this case we lose "Call provided event in scope and wait for finishing all the triggered effect." from `allSettled` although on backend we should wait for all asyncs.

**Comment by Bacher** [comment]
> But we can return value with type {status: 'done', value: Done} | {status: 'fail', value: Fail}
Actually I think your variant is very good. Semantic is really good, no errors, but details.

**Comment by zerobias** [comment]
Okay, then we'll implement that in next minor release

**Comment by zerobias** [comment]
Updates: this feature was implemented [in runtime](https://github.com/zerobias/effector/commit/17f44a1178c09fdcd1de20b8a012616e29cc871b) and [in typings](https://github.com/zerobias/effector/commit/fe24656ee74a97a57aa80455a21772348db849a2), the last question is what this method should return when used with events? Is it ok to return `void`? Or if we decided to return similar object, which `status` should be used? Can we add return object later?

After thinking about this for some time, I came to conclusion that calling `allSettled` with event should not return status object, at least it not useful in discussed case, but it is ok to introduce it later
